src/app/comments/add-comment/add-comment.component.html

```diff
@@ -163,7 +163,7 @@ <h5 class="modal-title">Member Sign-up</h5>
           <div class="form-control-container">
             <label class="control-label" >
               <input required type="checkbox" aria-label="Checkbox for row item" name="termsOfReference" [(ngModel)]='termsOfReference'>
-              By checking this box, I acknowledge that I have read, understood, and will abide by the <a href="https://projects.eao.gov.bc.ca/">Community Advisory Committee Terms of Reference</a>
+              By checking this box, I acknowledge that I have read, understood, and will abide by the <a href="https://www2.gov.bc.ca/assets/gov/environment/natural-resource-stewardship/environmental-assessments/guidance-documents/2018-act/community_advisory_committee_guideline_v1.pdf">Community Advisory Committee Terms of Referece</a>
```

Comment:
Not sure if it's required, but may want to add a target="_blank" so the PDF pops up in another tab without closing out the form in EPIC

Reply:
Also, misspelling of "Referece"

Reply:
Resolved